docker-compose.yml

```
@@ -19,4 +19,4 @@ services:
 networks:
   website-network:
     external: true
-    name: website-network
+    name: website-network
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)